app/core/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe object."""
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Ingredient(models.Model):
+    """Ingredient for recipes."""
+    name = models.CharField(max_length=255)
+    recipe = models.ForeignKey(
```

Comment:
This could also be a many-to-many relationship, so different recipes could share ingredients without duplicating the ingredients in the database.

Reply:
Ah just saw David also mentioned this, nothing from me then!

Reply:
Thanks, I replied on David's comment above. https://github.com/devashishsharma2302/tk-recipe-app/pull/1#discussion_r980857499

Reply:
Cool!